sound-equipment-fault-detection/vled/config.yaml

```diff
@@ -0,0 +1,9 @@
+grpc_server:
+  socket_path: /etc/kubeedge/vled.sock
+common:
+  name: VLED-mapper
+  version: v1.13.0
+  api_version: v1.0.0
+  protocol: virtualProtocol
```

Comment:
I have some questions about vled. Does this mapper manage virtual LED devices instead of real LED devices? So the device protocol here is virtualProtocol？

Reply:
Yes, LED is represented by a variable. If there is LED hardware, it can be replaced with the corresponding GPIO driver.
```shell
# LED is represented by a variable
const (
	LEDStatusOff int = 0
	LEDStatusOn  int = 1
)

var (
	vLED = LEDStatusOff // The default state is closed
	mu   sync.Mutex
)
```